src/drivers/dw/dma.c

```diff
@@ -439,7 +439,7 @@ static int dw_dma_stop(struct dma_chan_data *channel)
 	}
 
 	dcache_writeback_region(dw_chan->lli,
-				sizeof(struct dw_lli) * channel->desc_count);
+				sizeof(struct dw_lli) * (channel->desc_count + 2));
```

Comment:
@keyonjie looks like we need to make this a CONFIG option. ```CONFIG_DMA_DW_LLI_CACHE_LINE_SIZE```(force LLI to be cache line size)  and ```CONFIG_DMA_DW_LLI_PADDING``` (force LLI to be padded in LLI blocks). This way we can disable on legacy platforms and use only on Intel CAVS HW.

Reply:
@lgirdwood I didn't have intent to make this get merged to any branch, I will go forward to pursue explicit root cause of it.

Reply:
Thanks @keyonjie , we will also need a version for main branch.